libs/template/presets/src/b2c/experience/header.ts

```diff
@@ -34,7 +34,7 @@ export const HeaderTemplate: StaticComponent = {
           rules: [
             {
               layout: 'flex',
-              background: 'var(--oryx-color-primary-500)',
```

Comment:
Hm, is it a good idea to use direct color value?

Reply:
It's an exception; when we use the design token, we get a very ugly header with all colors in dark mode.